Slim/Player/TranscodingHelper.pm

```diff
@@ -355,7 +360,12 @@ sub getConvertCommand2 {
 	}
 	
 	# Test each profile in turn
-	PROFILE: foreach my $profile (@profiles) {
+	PROFILE: foreach (@profiles) {
+		my $instance = 0;
+	INSTANCE:
+		my $profile = $_ . ($instance++ ? '-' . ($instance-1) : '');
```

Comment:
Here again I think incrementing `$instance` on its own line wouldn't hurt the eye.

Reply:
Yes, I agree, I'm pushing the enveloppe of compactness too much here